projects.html

```diff
@@ -33,7 +33,7 @@ <h2>Software</h2>
                     <div class="projects-li-content">
                         <div class="project-header">
                             <a href="https://github.com/fmenozzi/shadertoy-rs">
-                                <h4 class="projects-project-title">shadertoy-rs</h4>
```

Comment:
These should all be h3's. You can't skip from h2 --> h4. I updated the style on this page to make the h3 look in line with h4. :)